doc/lsst.ip.diffim/AL_implementation.rst

```diff
@@ -0,0 +1,73 @@
+################################
+Alard-Lupton implementation code
+################################
+
+This page extends the API documentation of the implementation code.
+
+Solving for the matching kernel starts with the generation of the
+basis functions. The number of basis functions and the properties of
+the Gaussian bases (e.g. widths in pixels) are determined in
+`lsst.ip.diffim.makeKernelBasisList.generateAlardLuptonBasisList`. The
+decision logic depends on configuration and psf fwhm values of the
+images, some details is given in the function API documentation.
+
+:numref:`figclass` shows the main functional relation among the implementation
+classes. This diagram does not detail the abstraction hierarchy of the
+classes, they all have more general base classes. Please refer to the
+API documentation for details. There are also additional enums and
```

Comment:
Add a link to doxygen or whatever other API documentation you are referring to here

Reply:
OK to omit link for now as it may change